si/orange-canvas.yaml

```diff
@@ -0,0 +1,3536 @@
+gui/test.py:
+    class `QCoreAppTestCase`:
+        def `setUpClass`:
+            orangecanvas.testing: false
+            biolab.si: false
+        def `singleShot`:
+            Callable[[], Any]: false
+gui/toolbox.py:
+    ToolBox: false
+    _ToolBoxPage: false
+    index: false
+    widget: false
+    action: false
+    button: false
+    class `ToolBoxTabButton`:
+        def `__init__`:
+            font: false
+            QAbstractButton: false
+    class `ToolBox`:
+        Exclusive tabs: false
+        def `__init__`:
+            toolbox-scroll-area: false
+            toolbox-contents: false
+            toolbox-tab-action-group: false
+        def `createTabButton`:
+            toolbox-tab-button: false
+        def `__onTabActionToggled`:
+            action: false
+    A: false
+    B: false
+    C: false
+gui/toolgrid.py:
+    ToolGrid: false
+    _ToolGridSlot: false
+    button: false
+    action: false
+    row: false
+    column: false
+    class `ToolGridButton`:
+        def `__init__`:
+            darwin: false
+            QWidget: false
+        def `__textLayout`:
+            ' ': false
+            \n: false
+            '&': false
+            '&&': false
+    class `ToolGrid`:
+        def `__init__`:
+            sizePolicy: false
+        def `createButtonForAction`:
+            tool-grid-button: false
+gui/tooltree.py:
+    ToolTree: false
+    FlattenedTreeItemModel: false
+    class `ToolTree`:
+        def `__init__`:
+            tool-tree-view: false
+gui/utils.py:
+    def `disabled`:
+        setEnabled: false
+        isEnabled: false
+        '%r does not have ''enabled'' property': false
+    def `is_transparency_supported`:
+        win32: false
+        cygwin: false
+        darwin: false
+        linux: false
+        freebsd: false
+    def `is_x11_compositing_enabled`:
+        isCompositingManagerRunning: false
+    def `windows_set_current_process_app_user_model_id`:
+        nt: false
+    def `macos_set_nswindow_tabbing`:
+        darwin: false
+        .: false
+        libobjc: false
+        AppKit: false
+        NSWindow: false
+        'setAllowsAutomaticWindowTabbing:': false
+    def `css_gradient`:
+        \n: false
+        '    stop: {0:f} {1}': false
+        qlineargradient(\n: false
+        '    x1: {x1}, y1: {y1}, x2: {x2}, y2: {y2},\n': false
+        '{stops})': false
+    def `message_critical`:
+        An unexpected error occurred.: Prišlo je do nepričakovane napak
+        Error: Napaka
+    def `message_warning`:
+        Death could come at any moment.: Smrt pride nepričakovano.
+        Murphy lurks about. Remember to save frequently.: Murphy je vedno na preži. Ne pozabi na pogosto shranjevanje.
+        Warning: Opozorilo
+    def `message_information`:
+        Information: Informacije
+        I am not a number.: Nisem številka.
+    def `message`:
+        Message: Sporočilo
+        I am neither a postman nor a doctor.: Nisem ne poštar ne dohtar.
```

Comment:
Prosim, razloži :D

Reply:
Ta modul skbri za prikaz nekih sporočil na dnu widgeta. Istočasno pa vsebuje tudi kodo, s katero ga lahko stestiraš  - tako lahko nekdo, ki bi kaj spreminjal ta kos programa, požene to reč in vidi, ali izgleda, kot mora. Tega torej nihče nikoli ne bo videl in Aleš  se je malo zafrkaval, ko je programiral. Zato sem jaz za hec še prevedel in to bolj za hec. :)